packages/gestalt/src/Text.js

```diff
@@ -13,33 +13,40 @@ const SIZE_SCALE = {
   lg: 3,
 };
 
+type Align = 'start' | 'end' | 'center' | 'justify' | 'forceLeft' | 'forceRight';
```

Comment:
These types are all unchanged. Extracted for use in add Flow types for the PropTypes below, removing suppressions

Reply:
Nit: Curious to know what the difference in output looks like when using https://github.com/reactjs/react-docgen. Ideally we would use react-docgen to automatically generate our prop table in our docs.

Reply:
It doesn't look like we currently use that tool…is that something you've had in mind for a future improvement?